src/harness/fourslashImpl.ts

```diff
@@ -137,6 +137,10 @@ namespace FourSlash {
         throw new Error("Operation should be cancelled");
     }
 
+    export function ignoreInterpolations(diagnostic: string | ts.DiagnosticMessage): FourSlashInterface.DiagnosticIgnoredInterpolations {
+        return { template: typeof diagnostic === "string" ? diagnostic : diagnostic.message } as FourSlashInterface.DiagnosticIgnoredInterpolations;

```

Comment:
You shouldn't need this cast anymore (which is why I brought up the symbol earlier)? The linter doesn't catch it because technically the raw object type isn't _exactly_ the same type.